<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditPeoples</class>
 <widget class="QDialog" name="DlgEditPeoples">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peoples Editor ID: ####</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="dep_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>236</height>
       </rect>
      </property>
      <attribute name="icon">
       <iconset resource="res/main.qrc">
        <normaloff>:/icon/gen/file-open</normaloff>:/icon/gen/file-open</iconset>
      </attribute>
      <attribute name="label">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2" colspan="4">
        <widget class="QLineEdit" name="dep_name"/>
       </item>
       <item row="11" column="5">
        <widget class="QLineEdit" name="dep_flat">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Flat</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="dep_house">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Street</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Sex</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>House</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Birth</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="4">
        <widget class="QLineEdit" name="dep_family"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Family</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Patron</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="4">
        <widget class="QLineEdit" name="dep_patron"/>
       </item>
       <item row="10" column="2" colspan="4">
        <widget class="QComboBox" name="dep_street"/>
       </item>
       <item row="8" column="2">
        <widget class="QComboBox" name="dep_sex">
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>лю</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>лќ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="13" column="2" colspan="4">
        <widget class="QComboBox" name="dep_region"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Region</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QDateEdit" name="dep_birth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="14" column="4">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>329</width>
        <height>158</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="label">
       <string>&gt; Document</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Depart</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Series</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="5">
        <widget class="QLabel" name="label_15">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Give</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>410</width>
        <height>186</height>
       </rect>
      </property>
      <attribute name="label">
       <string>&gt; Phone and Etc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Home Phone</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Registration</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="22" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>ID Link</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Cell Phone</string>
         </property>
        </widget>
       </item>
       <item row="22" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item row="21" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item row="21" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>NIF</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QComboBox" name="comboBox_6"/>
       </item>
       <item row="19" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Reg Dates</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <widget class="QDateEdit" name="dateEdit_3"/>
       </item>
       <item row="14" column="2">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Flag</string>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QComboBox" name="comboBox_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="19" column="3">
        <widget class="QDateEdit" name="dateEdit_4"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>89</height>
       </rect>
      </property>
      <attribute name="label">
       <string>&gt; Comment</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dep_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgEditPeoples</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
